### DISCUSSION TITLE
A button to delete all keyframes in the wiggle 2 menu

### DISCUSSION THREAD

**Discussion by sergioarii** [shteeve3d/blender-wiggle-2#65] [Ideas]
A button that allows us to delete all the keyframes from the Wiggle 2 panel would be great. I often want to edit the physics properties and I find that it doesn't let me edit them because they have a keyframe and I have to delete them by hand, bone by bone.
An option to lock them so that keyframes cannot be added by accident
![Screenshot_7](https://github.com/shteeve3d/blender-wiggle-2/assets/72995128/bcaf75fc-213a-4de5-99e8-15bae9b83604)

**Comment by schroef** [comment]
No item in blender has this. You either clean your action or delete them in any of the other spots.
